help/grandorgue.xml

```diff
@@ -6352,7 +6352,7 @@ only allows adjustments at pipe level.
       </para>
       <para>
 The samples in the format above may also be packed with wavpack
-http://www.wavpack.com/.  GO only supports the wavpack V4 format
+http://www.wavpack.com/. GO only supports the WavPack V4 format
```

Comment:
Actually, I've tested with the stock WavPack of Ubuntu 20.04 - which is 5.2.0 - and it works just fine so this line should perhaps be changed...

Reply:
It seems that according to https://www.wavpack.com/WavPack5PortingGuide.pdf the version 5 is compatible with version 4 so that line could be changed to: 
`http://www.wavpack.com/. GO supports both WavPack 4 and WavPack 5.`

Reply:
I'm not familiar with WavPack. The text is "GO only supports the wavpack V4 format without hybrid compression and it must include all metadata. A good starting point for the wavpack compression options is -x6". Should I change it to "GO supports both the WavPack V4 and WavPack V5 formats, without hybrid compression and it must include all metadata. A good starting point for the wavpack compression options is -x6"? Or "GO only supports the wavpack V4 and V5 formats, without hybrid compression and it must include all metadata. A good starting point for the wavpack compression options is -x6"? Or something else?

Reply:
@willeke1234 WavPack doesn't use V4 or V5 to designate versions. Therefore I think that just the relevant part about the supported WavPack formats should be changed. The whole section in the help could then read:

`The samples in the format above may also be packed with WavPack http://www.wavpack.com/. GO supports both the WavPack 4 and WavPack 5 formats, without hybrid compression and it must include all metadata. A good starting point for the WavPack compression options is -x6.`

As noted in the document I linked to above, the 4 version is completely built into the 5 version and changes are only done on top of it (thus making it possible to use the WavPack 5 version exactly the same way as the 4 version was).